cli/lsp/config.rs

```diff
@@ -402,21 +400,21 @@ impl WorkspaceSettings {
 #[derive(Debug, Clone, Default)]
 pub struct ConfigSnapshot {
   pub client_capabilities: ClientCapabilities,
-  pub enabled_paths: HashMap<Url, Vec<Url>>,
   pub excluded_paths: Option<Vec<Url>>,
   pub has_config_file: bool,
   pub settings: Settings,
+  pub workspace_folders: Vec<(ModuleSpecifier, lsp::WorkspaceFolder)>,
```

Comment:
I think this is just lsp workspaces?

Reply:
Yeah this is just another field copied over from `Config` to `ConfigSnapshot`.

We look at LSP workspace folders pretty much just to respect the different vscode settings which might be specified for each. But we only respect that for `enable`, `enablePaths` and `codeLens.test`. A `deno.json` is currently discovered in the first folder's root or specified in the first folder's settings and applied to all. We have to manually implement support for many scopes, it would have little to do with workspace folders since subfolders matter just as much.